src/main/java/org/testng/internal/annotations/AnnotationHelper.java

```diff
@@ -217,7 +219,7 @@ private static void finishInitialize(ConfigurationAnnotation result, IConfigurat
 //    for (Class<?> cls : classes) {
         while (null != cls) {
           boolean hasClassAnnotation = isAnnotationPresent(annotationFinder, cls, annotationClass);
-          Method[] methods = cls.getDeclaredMethods();
+          Method[] methods = getDeclaredMethods(cls);
```

Comment:
The name is perfect :+1: 

> As far as I can tell it only returns default methods if you call it on the interface, otherwise you have to do some mining.

Does it mean there is no simplier method on the java8 reflexion api to find default methods? :disappointed: (I looked for it but I didn't find something relevant)